src/infra/dev/api/oi.tpl.json

```diff
@@ -289,29 +289,634 @@
         "x-amazon-apigateway-integration" : {
+          "passthroughBehavior" : "when_no_match",
+          "contentHandling" : "CONVERT_TO_TEXT",
+          "type" : "aws_proxy",
           "httpMethod" : "POST",
           "uri" : "arn:aws:apigateway:${aws_region}:lambda:path/2015-03-31/functions/${client_registration_lambda_arn}/invocations",
-          "responses" : {},
+          "requestParameters": {
+            "integration.request.header.accept": "'*/*'"
+          },
+          "responses" : {
+            "201": {
+              "statusCode": "201",
+              "responseParameters": {
+                "method.response.header.content-type": "'application/json'"
+              }
+            },
+            "400": {
+              "statusCode": "400",
+              "responseParameters": {}
+            },
+            "401": {
+              "statusCode": "401",
+              "responseParameters": {}
+            },
+            "403": {
+              "statusCode": "403",
+              "responseParameters": {}
+            },
+            "405": {
+              "statusCode": "405",
+              "responseParameters": {}
+            },
+            "429": {
+              "statusCode": "405",
+              "responseParameters": {}
+            },
+            "500": {
+              "statusCode": "500",
+              "responseParameters": {}
+            }
+          }
+        },
+        "summary": "OpenID Connect Dynamic Registration endpoint",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "redirect_uris": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": [
+                      "https://client.example.org/callback",
+                      "https://client.example.org/callback2"
+                    ]
+                  },
+                  "response_types": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": ["code", "token_id_token"]
+                  },
+                  "grant_types": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": ["authorization_code", "implicit"]
+                  },
+                  "application_type": {
+                    "type": "string",
+                    "example": "web"
+                  },
+                  "contacts": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": ["user1@example.org", "user2@example.org"]
+                  },
```

Comment:
```suggestion

```

Reply:
did not get it, is it just a tabulation?

Reply:
No. These are fields not present Java-side

Reply:
ops, sorry